src/proto/EthereumAbi.proto

```diff
@@ -0,0 +1,295 @@
+syntax = "proto3";
```

Comment:
Missing doc for some of the params, it's important as the doc is generated

Reply:
Will add, thanks

Reply:
Done